lib/__tests__/harvest.test.ts

```diff
@@ -159,18 +159,6 @@ describe("Integration tests", () => {
         );
         expect(userToken).toEqual("user-id-1");
       });
-
-      it("should get _hasCredentials from the instance", async () => {
```

Comment:
This test is redundant. We do not need `_hasCredentials` anywhere in search-insights or InstantSearch.js. (The test may have been added to see the possibility of avoiding duplicated init calls, but it's taken care in another PR.)